chart/templates/hpa-external-metric.yaml

```diff
@@ -1,8 +1,12 @@
 {{- $root := . -}}
 {{- range $name, $service := $root.Values.web.webServices }}
 {{- if $service.autoscaling.enabled }}
-{{- $pool := "sum:puma.pool_capacity{kube_cluster_name:dsva-vagov-" }}
-{{- $threads := "sum:puma.max_threads{kube_cluster_name:dsva-vagov-" }}
+{{- $pool := "avg:puma.pool_capacity{kube_cluster_name:dsva-vagov-" }}
+{{- $pool_with_filters := "(" $pool $root.Values.target_env "-cluster,kube_deployment:" $name ")" }}
+{{- $threads := "avg:puma.max_threads{kube_cluster_name:dsva-vagov-" }}
+{{- $threads_with_filters := "(" $threads $root.Values.target_env "-cluster,kube_deployment:" $name ")" }}
+{{- $replicas_available := "sum:kubernetes_state.deployment.replicas_available{kube_cluster_name:dsva-vagov-" }}
+{{- $replicas_with_filters := "(" $replicas_available $root.Values.target_env "-cluster,kube_deployment:" $name ")" }}
```

Comment:
Could these two lines be combined?

Reply:
I think this is cleaner and easier to understand with them being separate. If we ever decide to change something, it seems more complex if we fold in the first query with the query_with_filters